src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorSettings.cs

```diff
@@ -64,6 +64,11 @@ public enum ConnectorCompatibility
         // Swagger File Conformity
         // - parameters appear in the order specified in the swagger file
         // - internal visible parameters are completely hidden (required/optional, with or without default value)
-        SwaggerCompatibility = 2
+        SwaggerCompatibility = 2,
+
+        // Swagger File Conformity for CDP connectors
+        // - same as Swagger File Conformity
+        // - does not require format="enum" for identifying enumerations
+        CdpCompatibility = 3
```

Comment:
>CdpCompatibility [](http://example.com/codeflow?start=8&length=16)

This really is a misnomer.  It's what we should have originally done for SwaggerCompat,  (but can't change swagger compat due to breaking changes). 
But not sure what a better name would be.

Reply:
Not sure what better name to use